packages/destination-actions/src/destinations/engage-messaging-twilio/sendSms/sms-sender.ts

```diff
@@ -23,6 +35,17 @@ export class SmsMessageSender extends MessageSender<Payload> {
   getExternalId = () => this.payload.externalIds?.find(({ type }) => type === 'phone')
 
   getBody = async (phone: string) => {
+    if (!this.payload.body && !this.payload.contentSid) {
+      this.logger?.error(
+        `TE Messaging: Unable to process SMS, no body provided and no content sid provided - ${this.settings.spaceId}`
+      )
+      throw new IntegrationError(
+        'Unable to process sms, no body provided and no content sid provided',
+        'Invalid parameters',
+        400
+      )
```

Comment:
A better error in this case will be the PayloadValidationErrror. Payload Validation error messages are shown like below in EventDelivery along with the error message.
<img width="1506" alt="image" src="https://user-images.githubusercontent.com/109586712/234486537-e250c8f1-5fad-4e48-a07b-9c42a6926e6f.png">


```suggestion
      throw new PayloadValidationError(
        'Unable to process sms, no body provided and no content sid provided'
      )
```